### PR TITLE
Create styling-report-items.md

### PR DESCRIPTION
**Pull request by DimitarNikolovv06**

### CODE REVIEW

**Comment by DimitarNikolovv06** [comment]
I added the case with the Graph data points' styling that Vankata mentioned to a Notes section, if you know of any other specific cases, feel free to share.

File: designing-reports/styling-reports/styling-report-items.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                    <li>Bold - <strong>Boolean</strong>. Indicates whether this Font is bold. </li>
+                    <li>Italic - <strong>Boolean</strong>. Indicates whether this Font is italic.</li>
+                    <li>Name - <strong>String</strong>. The face name of this Font.</li>
+                    <li>Size - <strong>Telerik.Reporting.Drawing.Unit</strong>. The size of this Font measured in Units.
+                    </li>
+                    <li>Strikeout - <strong>Boolean</strong>. Indicates whether this Font specifies a horizontal line
+                        through the font.</li>
+                    <li>Style - <strong>FontStyle</strong>
+                        enumeration that contains style information for this Font.</li>
+                    <li>Underline - <strong>Boolean.</strong>Indicates whether this Font is underlined.</li>
```
Comment (todorarabadzhiev) [inline designing-reports/styling-reports/styling-report-items.md:102]: Add space:
'<strong>Boolean.</strong>Indicates ' > '<strong>Boolean</strong>. Indicates '

**Review by todorarabadzhiev — CHANGES_REQUESTED** [review]
I think you should include the information that when you don't set a Styling property of a report item, it inherits the corresponding value from its parent.

"The Default property is used to set all borders widths at once." - Actually, this is also used when you don't set explicitly a border property. For example, if you set the Default and you set only the Top, then the Default will be applied to Bottom, Left, and Right only.

Instead of writing everywhere 'The default value is ...', consider adding this at the top or at the bottom of the cell with some styling, e.g. italic,  and the units if applicable like:
"default is '1 in'"
This way, it will be more visible.

File: designing-reports/styling-reports/styling-report-items.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+title: Styling Report Items
+page_title: Styling Report Items 
+description: Styling Report Items
```
Comment (nelci592) [inline designing-reports/styling-reports/styling-report-items.md:4]: You can consider different description. Maybe something like "The article provides the available properties that can be used for the report items styling."

**Review by nelci592 — APPROVED** [review]

**Pull request merged**